apns_suite_test.go

```diff
@@ -5,8 +5,46 @@ import (
 	. "github.com/onsi/gomega"
 
 	"testing"
+	"time"
 )
 
+type mockConn struct {
```

Comment:
is mockConn something that users of this library would find useful when testing?